src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -23,11 +25,15 @@ public static long countWords() throws IOException {
     }
 
     public static void printLongestWordTop100() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+        String contents = Files.readString(Paths.get("src/main/resources/fp/war-and-peace.txt"));
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(word -> word.length() > 12)
+                .sorted(Comparator.comparing(String::length, Comparator.reverseOrder()))
+                .limit(100)
+                .distinct()
```

Comment:
```suggestion
                .filter(word -> word.length() > 12)
                .sorted(Comparator.comparing(String::length, Comparator.reverseOrder()))
                .distinct()
                .limit(100)
```
요구사항 대로라면 중복을 제거하고 100개 limit을 거는 것이 맞을 것 같아요